docker-compose.yml

```diff
@@ -32,6 +32,56 @@ services:
   rosie:
     image: serenata/rosie
 
+  prometheus:
+    image: arthursens/serenata-prometheus
+    ports: 
+    - 9090:9090
+    links:
+    - node_exporter:node_exporter
+    - process_exporter:process_exporter
+    - postgres_exporter:postgres_exporter
+    - bi_exporter:bi_exporter
+
+  grafana:
+    image: arthursens/serenata-grafana
+    links:
+    - prometheus:prometheus
+    - postgres:postgres
+    ports:
+    - 3000:3000
+
+  node_exporter:
+    image: prom/node-exporter
+
+  process_exporter:
+    image: ncabatoff/process-exporter
+    privileged: yes
+    volumes: 
+    - /proc:/host/proc
+    - ./prometheus/process-exporter:/config
+    command: --procfs /host/proc -config.path /config/config.yml
+
+  postgres_exporter:
+    image: wrouesnel/postgres_exporter
```

Comment:
This agent is responsible to execute queries at our postgres and expose the results.
It already have some monitoring queries that are executed by default and more can be added with a YAML file.

I've added some monitoring queries [here](https://github.com/okfn-brasil/serenata-de-amor/pull/484/files#diff-5591516ea0bb28550071905892354650R1).

More information about this exporter can be found at [this repository](https://github.com/wrouesnel/postgres_exporter)